File: samples/server/openapi3/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/model/InlineObject1.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*
import io.swagger.annotations.ApiModelProperty

/**
 * 
 * @param additionalMetadata Additional data to pass to server
 * @param file file to upload
 */
data class InlineObject1 (

        @ApiModelProperty(example = "null", value = "Additional data to pass to server")
        @JsonProperty("additionalMetadata") val additionalMetadata: String? = null,

        @ApiModelProperty(example = "null", value = "file to upload")
        @JsonProperty("file") val file: java.io.File? = null
) {

}

File: samples/server/petstore/scala-akka-http-server/src/main/scala/org/openapitools/server/model/InlineObject1.scala
package org.openapitools.server.model

import java.io.File

/**
 * @param additionalMetadata Additional data to pass to server for example: ''null''
 * @param file file to upload for example: ''null''
*/
final case class InlineObject1 (
  additionalMetadata: Option[String],
  file: Option[File]
)

